enginetest/memory_harness.go

```diff
@@ -70,6 +71,12 @@ func NewMemoryHarness(name string, parallelism int, numTablePartitions int, useN
 		externalProcedureRegistry.Register(esp)
 	}
 
+	var useServer bool
+	if _, ok := os.LookupEnv("SERVER_ENGINE_TEST"); ok {
+		useServer = true
+	}
+	useServer = true
```

Comment:
This line was probably just for local testing, I'm guessing? Seems like otherwise it would always default to server mode 😉 

```suggestion
```